tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -183,20 +182,20 @@ def test_5_snapshots(
 
         time.sleep(2)
         logger.info("Create snapshot %s #%d.", snapshot_type, i + 1)
-        snapshot = microvm.make_snapshot(snapshot_type)
+        new_snapshot = microvm.make_snapshot(snapshot_type)
 
         # If we are testing incremental snapshots we must merge the base with
         # current layer.
         if snapshot.is_diff:
-            logger.info("Base: %s, Layer: %s", base_snapshot.mem, snapshot.mem)
-            snapshot = snapshot.rebase_snapshot(
-                base_snapshot, use_snapshot_editor=use_snapshot_editor
+            logger.info("Base: %s, Layer: %s", snapshot.mem, snapshot.mem)
```

Comment:
did we mean to print the same value twice?

Reply:
absolutely not, thanks for catching!